README.md

```diff
@@ -24,8 +24,15 @@ $ pip install -r requirements.txt
 [venv]: https://virtualenv.pypa.io/en/stable/
 [pip]: https://pip.pypa.io/en/stable/installing/
 
-Once the dependencies are installed, you can start a local server to view the
-docs:
+Once the dependencies are installed you need to ensure you have also installed
+ any submodules (i.e. source code for the docs is in a separate submodule) 
+using:
+
+```
+$ git pull --recurse-submodules
```

Comment:
I think's to update once you have the code. So you need to replace that line with:

```
git submodule update --init
```

The one you give should be used only when the source code (from the remote repository) has changed and you want to update your local copy of it.